cfgov/unprocessed/js/routes/on-demand/ask-autocomplete.js

```diff
@@ -3,13 +3,33 @@
    ========================================================================== */
 
 import Autocomplete from '../../molecules/Autocomplete';
+import Analytics from '../../modules/Analytics';
 
 const URLS = {
   en: '/ask-cfpb/api/autocomplete/?term=',
   es: '/es/obtener-respuestas/api/autocomplete/?term='
 };
 
 const autocompleteContainer = document.querySelector( '.m-autocomplete' );
+const errorMessage = document.querySelector( '#o-search-bar-error_message' );
+const submitButton = document.querySelector( '.o-search-bar button[type="submit"]' );
+
+/**
+ * Disable the submit button if the query character limit is reached
+ * @param {Object} event The maxCharacterChange event object dispatched from the
```

Comment:
I usually put a dash after the param name and add a period to the end, like:

```
* @param {Object} event - The maxCharacterChange event object dispatched from the
* autocomplete.
```

Reply:
Fixed in 608db2c